File: 알고리즘/권하경/20-06-02/20-06-02.js
function solution(board, moves) {
  var answer = 0;
  let arr = [];
  moves.forEach(element => {
    board.some(items => {
      if(items[element-1]!=0){
        arr.push(items[element-1]);
        items[element-1]=0;
        return 1;
      }
    });
    if(arr[arr.length-1]==arr[arr.length-2]&arr.length>1) {
      arr.pop();
      arr.pop();
      answer++;
    }
  });
  return answer*2;
}

File: 알고리즘/권하경/20-06-02/20-06-02.py
def solution(board, moves):
    answer = 0
    arr = []
    for element in moves:
      for items in board:
        if items[element-1]:
          arr.append(items[element-1])
          items[element-1]=0
          break
        
      if len(arr)>1 and arr[len(arr)-1]==arr[len(arr)-2]:
        arr.pop()
        arr.pop()
        answer+=1
                
    return answer*2